client/rest-high-level/src/main/java/org/elasticsearch/client/ml/OpenJobResponse.java

```diff
@@ -71,18 +88,22 @@ public boolean equals(Object other) {
         }
 
         OpenJobResponse that = (OpenJobResponse) other;
-        return isOpened() == that.isOpened();
+        return isOpened() == that.isOpened()
```

Comment:
> it would have to become configurable wrt `acknowledged` field name though

I don't think this would be a good idea.  We made a mistake really in not using `acknowledged` as the field in this class back in 2016.  We stuck to the Prelert heritage instead of making things consistent with the Elastic way.  Unfortunately now making it consistent would be breaking change, and it doesn't seem like it's a big enough problem to warrant the break.  But making it easier for others to be inconsistent in the future doesn't seem like a good idea.